net-misc/casync/casync-2_p20190226.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 inherit vcs-snapshot meson
```

Comment:
Remove vcs-snapshot usage, it's not really needed here since you can determine S reliably.

Reply:
Done.